plugins/chat/app/services/chat/create_thread.rb

```diff
@@ -30,50 +30,47 @@ class CreateThread
     policy :threading_enabled_for_channel
     model :original_message
     transaction do
-      step :find_or_create_thread
+      model :thread, :find_or_create_thread
       step :associate_thread_to_message
-      step :fetch_membership
-      step :publish_new_thread
-      step :trigger_chat_thread_created_event
     end
+    model :membership, optional: true
```

Comment:
Do we have tests to cover the scenario when membership does not exist? I'm wondering what will happen since it is optional.

Reply:
Actually, the tests cover the scenario with `membership` being `nil` :sweat_smile: I’m not sure we have a case where it’s not `nil` :thinking:

Reply:
This doesn’t seem to be used anywhere, I’ll remove it.